spec.go

```diff
@@ -438,6 +438,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
+		name = spec.Linux.CgroupsPath
+	}
```

Comment:
Also, since cgroupsPath does not support split hierarchy, would parent be ever different than what this line retrieves: https://github.com/opencontainers/runc/blob/master/spec.go#L436